corehq/warehouse/transforms/sql/user_group_dim.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO {{ user_group_dim }} (
+    domain,
+    user_dim_id,
+    group_dim_id,
+    dim_last_modified,
+    dim_created_on,
+    deleted
+)
+
+SELECT
+    ud.domain,
+    ud.id,
+    gd.id,
+    now(),
+    now(),
+    combined_user_group_table.deleted
+FROM
+(
+(
+    SELECT
+        UNNEST(user_ids) AS user_id,
+        group_id AS group_id,
+        false AS deleted
+    FROM
+        {{ group_staging }}
+)
```

Comment:
curious what you think of this query. i know subqueries like this aren't advisable, but i think this will be fast and small in almost all cases